tester/device.go

```diff
@@ -0,0 +1,58 @@
+package tester
+
+import (
+	qt "github.com/frankban/quicktest"
+)
+
+const maxRegisters = 200
+
+// I2CDevice represents a mock I2C device on a mock I2C bus.
+type I2CDevice struct {
+	C    *qt.C
+	Addr uint8
```

Comment:
I think I'd leave this unexported, and maybe define an exported `Addr` method instead.

Reply:
Done.